packages/cfd/src/Constants/cfd_compare_account_content.ts

```diff
@@ -181,15 +181,11 @@ export const cr_real_footer_buttons: TCompareAccountFooterButtonData[] = [
     { label: localize('Add'), action: 'financial_labuan' },
     { label: localize('Add'), action: 'derivx' },
 ];
-export const eu_real_footer_button: TCompareAccountFooterButtonData[] = [
+export const getEuFooterButtons = (): TCompareAccountFooterButtonData[] => [
     { label: localize('Add'), action: 'financial_maltainvest' },
 ];
 
-export const eu_demo_footer_button: TCompareAccountFooterButtonData[] = [
```

Comment:
why did you removed this?

Reply:
@mahdiyeh-deriv  eu-demo and eu-real have only one button. The button label ('Add') and the corresponding jurisdiction is same (maltainvest).
so i made them to a common function.

Reply:
got it, thanks. so ignore my suggestion.